modules/channel-telegram/src/renderers/carousel.ts

```diff
@@ -0,0 +1,45 @@
+import * as sdk from 'botpress/sdk'
+import path from 'path'
+import { TelegramContext } from 'src/backend/typings'
+import { CallbackButton, Markup } from 'telegraf'
+import Extra from 'telegraf/extra'
+
+export class TelegramCarouselRenderer implements sdk.ChannelRenderer<TelegramContext> {
+  getChannel(): string {
+    return 'telegram'
+  }
+
+  getPriority(): number {
+    return 0
+  }
+
+  getId() {
```

Comment:
?

Reply:
This isn't really used at the moment but I thought having ids for renderers could be useful. For example if you want to disable one or overidde one, etc. you could do it by using its id.

Reply:
How can it be overridden ?